src/backend/core/api/viewsets.py

```diff
@@ -479,6 +479,27 @@ def annotate_user_roles(self, queryset):
             user_roles=db.Value([], output_field=output_field),
         )
 
+    def annotate_has_owner_role(self, queryset):
```

Comment:
Can be removed now? I agree that it should be in the trashbin method if it is not factorized at least 3 times.